.github/workflows/link-validator.yml

```diff
@@ -0,0 +1,34 @@
+name: Link Validator
+
+on:
+  pull_request_target:
+    branches: [$default-branch]
+  schedule:
+    - cron:  '0 6 * * 1'
```

Comment:
How come not on a PR?

Reply:
`pull_request_target` means "run on pull requests targeting the listed branches".

Reply:
Got it.  Is it also running on a cron?

Reply:
Yes, every gardening Monday.